README.md

```diff
@@ -16,14 +16,14 @@ NodeJS: https://nodejs.org/en/download/
 
 MongoDB: https://docs.mongodb.com/manual/installation/
 
-You will need start a MongoDB background process. Refer to the official documentation for directions on how to set this up for your OS.
+You will need to either start MongoDB background process, or manually start and stop the process in a separate console. Refer to the official documentation for directions on how to set this up for your OS.
```

Comment:
Could we add a link to the official documentation?

Reply:
line 17